components/butterfly/src/server.rs

```diff
@@ -322,29 +322,29 @@ impl Server {
                 Ok(Server { name: Arc::new(name.unwrap_or_else(|| member_id.clone())),
                             // TODO (CM): could replace this with an accessor
                             // on member, if we have a better type
-                            member_id:            Arc::new(member_id),
-                            member:               Arc::new(RwLock::new(myself)),
-                            member_list:          Arc::new(MemberList::new()),
-                            ring_key:             Arc::new(ring_key),
-                            rumor_heat:           RumorHeat::default(),
-                            service_store:        RumorStore::default(),
+                            member_id: Arc::new(member_id),
+                            member: Arc::new(RwLock::new(myself)),
+                            member_list: Arc::new(MemberList::new()),
+                            ring_key: Arc::new(ring_key),
+                            rumor_heat: RumorHeat::default(),
+                            service_store: RumorStore::default(),
                             service_config_store: RumorStore::default(),
-                            service_file_store:   RumorStore::default(),
-                            election_store:       RumorStore::default(),
-                            update_store:         RumorStore::default(),
-                            departure_store:      RumorStore::default(),
-                            swim_addr:            swim_socket_addr,
-                            gossip_addr:          gossip_socket_addr,
-                            suitability_lookup:   Arc::new(suitability_lookup),
-                            data_path:            data_path.as_ref().map(|p| p.into()),
-                            dat_file:             None,
-                            departed:             Arc::new(AtomicBool::new(false)),
-                            pause:                Arc::new(AtomicBool::new(false)),
-                            swim_rounds:          Arc::new(AtomicIsize::new(0)),
-                            gossip_rounds:        Arc::new(AtomicIsize::new(0)),
-                            block_list:           Arc::new(RwLock::new(HashSet::new())),
-                            socket:               None,
-                            election_timers:      Arc::new(Mutex::new(HashMap::new())), })
```

Comment:
This seems like an odd change in formatting.  I thought we had `rustfmt` configured to vertically line up the fields.

Reply:
Only when they're all of the form `foo: bar`, apparently. `suitability_lookup` now breaks that pattern.